packages/tasit-action/src/TasitAction.js

```diff
@@ -168,22 +168,29 @@ class TransactionSubscription extends Subscription {
           this.#tx.hash
         );
 
-        if (receipt !== null) {
-          this.#txConfirmed = true;
-        } else {
-          if (this.#txConfirmed)
-            this._emitErrorEvent(
-              new Error(`Your message has been included in an uncle block.`),
-              eventName
-            );
+        const includedInAnUncle =
+          (receipt === null && this.#txConfirmations > 0) ||
+          (receipt !== null && receipt.confirmations < this.#txConfirmations);
 
+        if (includedInAnUncle) {
+          this._emitErrorEvent(
+            new Error(`Your message has been included in an uncle block.`),
```

Comment:
In the README for this functionality we'll want to clearly document what we expect the users to do in response to an event like this in their UI

Reply:
Do you want that I create an issue about this?

Reply:
Sure, thanks. You can assign it to me

Reply:
https://github.com/tasitlabs/TasitSDK/issues/129

Reply:
Thanks! Resolving this.